engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -131,7 +129,10 @@ class K8sDeploymentManager(modelData: ModelData, config: K8sDeploymentManagerCon
     )
   }
 
-  protected def deploymentForData(processVersion: ProcessVersion, configMapId: String): Deployment = DeploymentPreparer(processVersion, config, configMapId)
+  protected def deploymentForData(processVersion: ProcessVersion, configMapId: String): Deployment = {
+    val deploymentPreparer = DeploymentPreparer(config)
```

Comment:
preparer can be created on class level now

Reply:
done